client/lib/gsuite/get-gsuite-subscription-status.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Gets Google Workspace subscription status for a given domain object,
+ * If resulting status is null or undefined, it returns empty string.
```

Comment:
I think we could remove that line as it just duplicates what's already written in the `@returns` param below.